lib/tower_cli/resources/credential.py

```diff
@@ -144,6 +144,8 @@ class Resource(models.Resource):
     vault_password = models.Field(password=True, required=False)
 
     def create(self, **kwargs):
+        """Create a credential.
+        """
```

Comment:
Indeed. I have swept through the help texts and this missing issue should be solved altogether after merging.